### ISSUE TITLE
Config file 

### ISSUE THREAD

**Issue by AEK-BKF** [thekordy/ticketit#181]
Hi,
first of all thanks a lot for this nice package,
I think it will be very nice to add a config file to set :
-> Path to User model, (here App\User, so we can set App\Models\User instead )
-> Name attribute, ( we can use username ..)
-> Settings table, (we can rename it to configs for example )

Please could you check these points ? thanks.

**Comment by thekordy** [comment]
Yes, I am thinking in a new version focused on many things including the points you mentioned.

**Comment by AEK-BKF** [comment]
Thanks a lot :+1: 
We're waiting for you :)

**Comment by maximusblade** [comment]
Perhaps I missed something.
Is there supposed to be a config file in my config?
I got the packaged "working"
But I'm unable to successfully extend **controllers** and **models** ?
- I would like to add some extra functionality to the Controllers.
- Is this related to my inability to view my own open tickets as admin?

**Comment by balping** [comment]
According to my knowledge, now extending controllers and models is not possible. And there are no config files, all settings are stored in the db.

But.

This package is currently being rewritten from zero, check [this branch](https://github.com/thekordy/ticketit/tree/2.0) It will be very flexible and settings will be stored in config files. However its development is still in early stages.

So since 0.2 will be discontinued as 2.0 is released, I suggest you to fork this package and modify controllers as you need to. Or there's always the dirty vendor code overwriting method
